src/context.ts

```diff
@@ -26,36 +26,41 @@ export interface ICommonContext {
 	getHashId: (originString: string, originIsNotUnique?: boolean) => string
 	/** Un-hash, is return the string that created the hash */
 	unhashId: (hash: string) => string
+
+	logDebug: (message: string) => void
+	logInfo: (message: string) => void
+	logWarning: (message: string) => void
+	logError: (message: string) => void
 }
 
-export interface NotesContext extends ICommonContext {
-	error: (message: string, params?: { [key: string]: any }) => void
-	warning: (message: string, params?: { [key: string]: any }) => void
+export interface IUserNotesContext extends ICommonContext {
+	userError: (message: string, params?: { [key: string]: any }) => void
+	userWarning: (message: string, params?: { [key: string]: any }) => void
 }
 
 /** Studio */
 
-export interface IStudioConfigContext {
+export interface StudioContext extends ICommonContext {
```

Comment:
While we're at it, should we make an effort for consistent interface naming? At least in this file? (`ICommonContext` vs `CommonContext`)